src/components/Project/ImageAnalysis/LinePlot.vue

```diff
@@ -1,13 +1,10 @@
 <script setup>
-import { onMounted, onUnmounted, ref, computed, watch } from 'vue'
-import { useSettingsStore } from '@/stores/settings'
+import { onMounted, ref, watch } from 'vue'
 import * as d3 from 'd3'
 
 const svg = ref(null)
-const store = useSettingsStore()
 
-const randomNumbers = computed(() => store.randomNumbers)
-const lineLength = computed(() => store.lineLength)
+const props = defineProps(['yAxisData', 'xAxisSize'])
```

Comment:
nit-picking here, I think these could be named more clearly. Maybe something like `luminosity` and `arcsecs` or if you want to include the y and x values to make it knowns what's what, `yAxisLuminosity` and `xAxisArcsecs`

Reply:
agreed, since this plot is tuned for the line analysis I went with your second suggestion. In the future if we have another graph though maybe we can add two more props to this component yAxisName and xAxisName so we can reuse it.